diff --git a/dist/index.js b/dist/index.js
index c8cbe267d2cfa3268727faeb1f06c8cc6ba7155a..953b82b8d95cadf6d1f6a7fa51894aad8aab29ce 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -5969,7 +5969,7 @@ const RemovePluginMetadataPlugin = (nuxt) => createUnplugin(() => {
 					map: null
 				};
 			}
-			if (!findExports(code).find((e) => e.type === "default" || e.name === "default")) {
+			if (!findExports(code).find((e) => e.type === "default" || e.names.includes("default"))) {
 				pluginDiagnostics.NUXT_B2005({ src: plugin.src });
 				return {
 					code: "export default () => {}",
diff --git a/dist/index.mjs b/dist/index.mjs
index c8cbe267d2cfa3268727faeb1f06c8cc6ba7155a..953b82b8d95cadf6d1f6a7fa51894aad8aab29ce 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -5969,7 +5969,7 @@ const RemovePluginMetadataPlugin = (nuxt) => createUnplugin(() => {
 					map: null
 				};
 			}
-			if (!findExports(code).find((e) => e.type === "default" || e.name === "default")) {
+			if (!findExports(code).find((e) => e.type === "default" || e.names.includes("default"))) {
 				pluginDiagnostics.NUXT_B2005({ src: plugin.src });
 				return {
 					code: "export default () => {}",
